src/instructlab/model/linux_test.py

```diff
@@ -0,0 +1,73 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Standard
+from pathlib import Path
+from typing import Optional
+import logging
+import os
+
+# Third Party
+from datasets import load_dataset
+from openai import OpenAI, Stream
+from tqdm import tqdm
+import click
+
+# Local
+from .. import config
+from ..utils import get_sysprompt
+
+
+def linux_test(
```

Comment:
it looks like this would work cross platform?

Reply:
May be.